src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
This change seems to remove some significant functionality, illustrated by two use cases in particular:

1. This regresses from BAM for the use case of reading all reads in coordinate sorted order for a `ReadGroupSet`, or a single coordinate sorted BAM file. My concerns would be satisfied if there were an additional option to query by `readGroupSetId`.

2. Loss of the ability to do "multi-BAM slicing" on the server side. Of course there is a cost of implementation, but I do believe this cost was considered when defining the SearchReads semantics. I would be curious to know if opinions have changed on the subject.

In Google's implementation of v0.5.1, we support the readGroupIds field but we make the additional requirement that all provided read group IDs belong to the same read group set. We recommend querying by readGroupSetId instead (and we support searching by multiple IDs).